### PR TITLE
Easier install, deprecate gpg/aes pipelines, notify about directories

### PR DESCRIPTION
**Pull request by leondutoit**
Closes https://github.com/unioslo/tsd-api-client/issues/50 and https://github.com/unioslo/tsd-api-client/issues/49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 
 ## Install
 
-Download the [latest release](https://github.com/unioslo/tsd-api-client/releases), unzip the archive, and then install as follows:
-
 ```bash
-cd tsd-api-client
-pip3 install -r requirements.txt
-python3 setup.py install
+pip3 install https://github.com/unioslo/tsd-api-client/archive/v2.4.0.zip
```
Comment (haatveit) [inline README.md:8]: I like the simplification but does GitHub really not provide a way to link the latest tag's tarball/zip? That's inconvenient.

Comment (leondutoit) [reply README.md:8]: will check it out

Comment (leondutoit) [reply README.md:8]: Looks like they do not :|

Comment (leondutoit) [reply README.md:8]: Will just have to live with updating the readme with new releases then. Guess we should eventually publish this on pypi.

**Pull request merged**